README.md

```diff
@@ -9,9 +9,10 @@ This repository is intended to collect guidance, documents, examples, templates
 * [Participating in Open Source Communities](https://www.linuxfoundation.org/resources/open-source-guides/participating-in-open-source-communities)
 * [OpenSSF Best Practices Badge Program](https://www.bestpractices.dev/en)
 * [Code for Science and Society's Digital Infrastructure Incubator resources](https://www.codeforsociety.org/incubator/resources/dii-resource)
-* [Mozilla's Open Leadership Training Series's Best Practices in Working Open](https://mozilla.github.io/open-leadership-training-series/)
```

Comment:
I think we can keep this, but change it to point to the archived version, as I believe the info is still useful. https://web.archive.org/web/20241028115543/https://mozilla.github.io/open-leadership-training-series/